sumSquareDifference.js

```diff
@@ -0,0 +1,24 @@
+class SumOfASquare {
+  constructor(num) {
+    this.num = num;
+  }
+  sumOfSquares() {
+    let arr = [];
+    for (let i = 1; i < this.num + 1; i++) {
+      arr.push(Math.pow(i, 2));
+    }
+    return arr.reduce((total, arr) => total + parseInt(arr), 0);
+  }
+  squareOfTheSums() {
+    let arr2 = [];
+    for (let i = 1; i < this.num + 1; i++) {
+      arr2.push(i);
+    }
+    return Math.pow(
+      parseInt(arr2.reduce((total, arr2) => total + parseInt(arr2), 0)),
```

Comment:
Instead of using an array and a reduce here. Could you simplify this code so that you're doing the summation as you go through the for loop?